plugins/azure/virtualmachines/computeGalleryRbac.js

```diff
@@ -0,0 +1,51 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure/');
+
+module.exports = {
+    title: 'Compute Gallery RBAC Sharing',
+    category: 'Virtual Machines',
+    domain: 'Compute',
+    description: 'Ensures that the Azure Compute Gallery is using RBAC only.',
+    more_info: 'As the Azure Compute Gallery, definition, and version are all resources, they can be shared using the built-in native Azure Roles-based Access Control (RBAC) roles. A direct shared gallery can\'t contain encrypted image versions. Community galleries can be used by anyone with an Azure subscription making the resource more vulnerable.',
```

Comment:
```suggestion
    more_info: 'As the Azure Compute Gallery, definition, and version are all resources, they can be shared using the built-in native Azure Roles-based Access Control (RBAC) roles. A direct shared gallery can\'t contain encrypted image versions. Community galleries can be used by anyone with an Azure subscription making the resource more vulnerable.',
```
'As the Azure Compute Gallery, definition, and version are all resources' seems confusing. Also the last two lines could probably be rephrased better